kolibri/utils/cli.py

```diff
@@ -422,11 +424,12 @@ def services(daemon=True):
     # Daemonize at this point, no more user output is needed
     if daemon:
 
+        logger.info("Going to daemon mode, logging to {0}".format(server.KOLIBRI_LOG))
```

Comment:
Best to just directly import the settings here and reference the logging filename.